core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -93,6 +94,8 @@ oppia.directive('audioTranslationBar', [
           $scope.checkingMicrophonePermission = false;
           $scope.audioTimerIsShown = true;
           $scope.audioIsCurrentlyBeingSaved = false;
+          $scope.disallowed = !GLOBALS.can_translate;
```

Comment:
Make the variable name a bit more descriptive like `$scope.translationIsDisallowed`.

Reply:
Ok done